src/shared/components/pipeline-run-logs/PipelineRunLogs.tsx

```diff
@@ -98,6 +99,11 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
         namespace: obj.metadata.namespace,
         isList: false,
       };
+
+    const waitingForPods = !!(activeItem && !resource);
+    const taskName = get(taskRunFromYaml, [activeItem, 'pipelineTaskName'], '-');
+    const pipelineRunFinished = ![runStatus.Running].includes(pipelineRunStatus);
```

Comment:
Why check against inclusion in an array?

Reply:
Initially had multiple statuses to check, but later realised its not required and missed to refactor this. Updated now